jupyterhub/jupyterhub/base/jupyterhub-dc.yaml

```diff
@@ -93,3 +93,12 @@ spec:
         name: config
   triggers:
   - type: ConfigChange
+  - type: ImageChange
+    imageChangeParams: 
+      automatic: true
+      containerNames:
+        -wait-for-database
```

Comment:
Currently this is being parsed as a string, this needs a space after the `-` to make it a list